### ISSUE TITLE
Chunk Loader not loading End chunks

### ISSUE THREAD

**Issue by Benjamin-deVries** [xJon/The-1.12.2-Pack#886]
ChickenChunks Chunk Loader does not appear to be loading chunks in the End.
I have an auto-enderaman farm connected to a Quantum Entangloporter that works fine when i am in the End at the farm, but fails when i am not.

I have not tried loading Nether chunks, but Overworld chunks load just fine.

**Comment by xJon** [comment]
I cannot reproduce this - are you playing on a specific server?
Are you sure your farm is within range of the chunk loader?

**Comment by Benjamin-deVries** [comment]
No, playing locally on my machine, not other mods/custom configuration.
Yes, please see the attached image of my setup
![Untitled](https://github.com/xJon/The-1.12.2-Pack/assets/70020583/51c0190f-98ad-49e1-a3b1-6856c64f12f6)

**Comment by xJon** [comment]
Are you sure the problem is with chunk loading?
For example, if you have a hopper connected to an ender chest, does that work when you're not in the area?
